README.md

```diff
@@ -24,11 +24,17 @@ This plugin has been created by the [GEM Foundation](http://www.globalquakemodel
 This plugin runs with [QGIS 3.0 and above](http://qgis.org/en/site/forusers/alldownloads.html)
 and above.
 
-On Microsoft Windows and Mac OS X, QGIS includes all the software dependencies needed by the plugin.
+On Microsoft Windows, QGIS includes all the software dependencies needed by the plugin.
 
-On Linux, please make sure that [Scipy](https://www.scipy.org/install.html) and
+On macOs and Linux, please make sure that [Scipy](https://www.scipy.org/install.html) and
 [Matplotlib](https://matplotlib.org/users/installing.html) with Qt4 backend are installed.
 
```

Comment:
I think the correct spelling is `macOS`.

Reply:
Yes, thanks, I actually wanted to type macOS...